autopush/settings.py

```diff
@@ -163,9 +178,28 @@ def __init__(self,
             self.endpoint_hostname,
             endpoint_port
         )
+
+        # not accurate under autoendpoint
```

Comment:
Sorry, this is basically a TODO since I'll be touching Settings next. Some settings are specific to endpoint or connection, e.g. "router_url" -- only used in websocket and produces a bogus value under the endpoint. I'm adding ws_url in the same manner, for now.

I will be cleaning that up -- pretty tempted to separate the Settings per endpoint/connection to follow suit w/ the App objects (test_integration's already half way there)